packages/clay-charts/demos/index.html

```diff
@@ -370,7 +399,18 @@ <h2>Bubble</h2>
 						document.querySelector('#vertical-small'),
 					];
 				}
-			}
+			},
+			legend: {
+				contents: {
+					bindto: '.pie-legend',
+					template(title, color) {
```

Comment:
Hey @julien 

Not sure if this is applicable or not, but should we consider bringing this svg template over to Clay somehow so that it can be leveraged? Maybe even if it was just provided as a utility.

Reply:
Yes, it sounds like a good idea

Reply:
@Robert-Frampton, should I close this pull request until we have an idea of how we want to include this svg template?

Reply:
Hey @julien, since this is adding missing config, let's merge it and then do the abstraction in another issue.